src/resources/postcss/tribe-common-admin.pcss

```diff
@@ -2118,6 +2139,84 @@ input[type="email"].tribe-events-admin-card__input {
 	}
 }
 
+.tribe-events-admin-notice {
+	background-color: #3D54FF;
+	height: 65px;
+
+	.tribe-events-admin-content-wrapper {
+		padding-top: 8px;
+		padding-bottom: 0;
+	}
+
+	p {
+		color: #fff;
+		display: inline-block;
+		font-family: Helvetica;
+		font-size: 16px;
+		line-height: 18px;
+		margin-top: 0px;
+		padding-bottom: 12px;
+		padding-left: 16px;
+		vertical-align: middle;
+		width: calc(100% - 60px);
+	}
+}
+
+.tribe-events-admin-notice__logo {
+	display: inline-block;
+}
+
+/* -- Event Tickets tweaks for welcome screen -- */
+.tribe-events-admin-tickets {
```

Comment:
tickets-admin css wasn't loading on the welcome screen. Not sure why, but for expediency, I put these back in common.